### DISCUSSION TITLE
SCAN CONNECT NOT WORKING

### DISCUSSION THREAD

**Discussion by EECvision** [WalletConnect/walletconnect-monorepo#3444] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/63562663/2f8db3ca-4a6c-475e-a52b-1cfa600c9dd0)

Hi, I am trying to connect to my dApp using my mobile metamask wallet. Scanning the QR code from the app does nothing. 

here is my code:
```
    try {
      const provider = await EthereumProvider.init({
        projectId: "4462f211ae9d8e31c96b53bd082c3ac0",
        chains: [1],
        optionalChains: [5, 80001],
        showQrModal: true,
        methods: [
          "eth_sendTransaction",
          "eth_signTransaction",
          "eth_sign",
          "personal_sign",
          "eth_signTypedData",
        ],
        events: ["chainChanged", "accountsChanged"],
        rpcMap: {
          80001: "https://rpc.ankr.com/polygon_mumbai",
        },
      });

      await provider.enable();
    } catch (error) {
      console.log(error);
    }
    
```

**Comment by glitch-txs** [comment]
You are using unsupported methods, try removing methods from config you will get some default automatically (personal_sign and eth_sendTransaction)

**Reply by EECvision** [reply]
Thank you for your response. I have tried it again without including methods and nothing happened.

**Reply by glitch-txs** [reply]
Have you tried with our examples?
https://github.com/WalletConnect/web3modal-examples/tree/main/walletconnect-modal-ethereum-provider-react

Are you getting any errors?
